feat(sig-services): Add easy interaction guideline (#229)
open-services-group/community: sig-services/service-requirements/easy-interaction.md

---
title: Easy to interact with a service
persona: Service owner, service user
phase: Service development
---

> Service should be simple to use. If the process in the service is complicated, it should utilize automation to make it easier (for example, installation). If the process needs some time to finish, it should inform the user about its state. Service should assume a reasonable standard configuration that users can easily modify. Interaction patterns should be making the learning curve shallow, meaning the expertise needed for the user to operate the service should not need to be on the same level as the maintainer.

Author: Resolved in latest commit.

## Requirements

- Service should be easy to use
- Complex processes should be simplified using automation

> - Default configuration should be reasonable and easy to modify

Author: Resolved good thought :+1:

- Service should use good interaction patterns to make the learning curve shallow
- Expertise needed to operate service for the user should be low

## Reasoning

> It is making service easy to use leads to users being able to learn how to use the service effectively faster. It also leads to higher user satisfaction because they don't have to review documentation to utilize basic functionality.

Reviewer: ```suggestion
The guidelines help make the service easy to use and leads to users being able to learn how to use the service effectively faster. It also leads to higher user satisfaction because they don't have to review the documentation to utilize basic functionality.
```

Author: Resolved in latest commit
